.pre-commit-config.yaml

```diff
@@ -18,3 +18,10 @@ repos:
       - id: ruff
         args: [ --fix ]
       - id: ruff-format
+  - repo: local
```

Comment:
Do we need to add anything in the Readme around how Trufflehog works and how to use locally?